automation/jleechanorg_pr_automation/automation_safety_manager.py

```
@@ -840,15 +840,15 @@ def _send_notification(self, subject: str, message: str) -> bool:
             msg = MIMEMultipart()
             msg["From"] = from_email
             msg["To"] = to_email
-            msg["Subject"] = f"[WorldArchitect Automation] {subject}"
+            msg["Subject"] = f"[Your Project Automation] {subject}"
 
             body = f"""
 {message}
 
 Time: {datetime.now().isoformat()}
```

Ruff (0.15.6)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)